code/game/machinery/firealarm.dm

```diff
@@ -323,8 +327,14 @@ FIRE ALARM
 		else
 			overlays += image('icons/obj/monitors.dmi', "overlay_green")
 
+	myArea = get_area(src)
```

Comment:
Is there a reason this is done on New level? machinery.dm also sets myArea in the Initialise